build.sbt

```diff
@@ -140,10 +138,23 @@ lazy val root: Project = (project in file("."))
 lazy val configTools = (project in file("configTools"))
   .enablePlugins(SbtTwirl)
   .settings(
-    addCompilerPlugin(
-      "org.typelevel" %% "kind-projector" % "0.13.3" cross CrossVersion.full
-    ),
     commonSettings,
+    crossScalaVersions := Seq(
+      "2.13.16",
+      scalaVersion.value
+    ),
+    /*
+     * The release process tries to find the previously released version
+     * so that it can check for the level of compatibility
+     * to determine the version number of the next release.
+     * If there is no previous release, the process breaks.
+     * To get around this we temporarily use the latest 2.13 release make the comparison.
+     *
+     * TODO: remove this when we have released a Scala 3 version
+     */
+    mimaPreviousArtifacts := Set(
+      organization.value %% name.value % "4.0.0" cross CrossVersion.for3Use2_13
+    ),
```

Comment:
Ah, this is an interesting approach! 

This problem - of the MIMA compatibility check failing when new artifacts are introduced (because there are no previous artifacts to compare against for compatibility testing) is a known and annoying issue:

* https://github.com/guardian/gha-scala-library-release-workflow/issues/33

...in the past, I've dealt with this simply by _disabling_ the automated compatibility testing when releasing new artifacts. Again, it needs to be reinstated after the new artifact is released.